css-positioning/absolute.css

```diff
@@ -0,0 +1,42 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  top: 6px;
+  right: 6px;
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+
+  /* your code here */
+  position: absolute;
+  bottom: 5px;
+
+  .overlay {
+    background: linear-gradient(transparent, rgba(0, 0, 0, 0.75));
+
+    /* your code here */
+  }
```

Comment:
Your `overlay` is currently not visible, you will need to set a height and width to make it actually visible. Also, this rule somehow got nested inside your `.joke` rule, please adjust that.